monitor/webserver/nginx_landing_page.sls

```diff
@@ -0,0 +1,60 @@
+
+nginx:
+  pkg:
+    - installed
+
+openssl:
+  pkg:
+    - installed
+
+/usr/share/nginx/html/id:
+  file:
+    - managed
+    - makedirs: true
+    - contents: '{{ grains.id }}'
+    - require:
+      - pkg: nginx
+
+/usr/share/nginx/html:
+  file:
+    - recurse
+    - source: salt://monitor/var/www/html
+    - template: jinja
+    - makedirs: true
+    - require:
+      - pkg: nginx
+
+/etc/nginx/conf.d/:
```

Comment:
Since on other hosts that is where the conf lived is why I moved it there. I cleaned out the sites-enabled to prevent confusion. Should I shift it back?